linkml_runtime/utils/schemaview.py

```diff
@@ -173,7 +186,7 @@ def _closure(
     :rtype: list[str | ElementName | ClassDefinitionName | EnumDefinitionName | SlotDefinitionName | TypeDefinitionName]
     """
     if kwargs and kwargs.get("detect_cycles"):
-        detect_cycles(f, x)
+        detect_cycles(f, [x])
```

Comment:
currently this is the only place in the code where `detect_cycles` is called directly